spec/classes/cli_plugins_spec.rb

```diff
@@ -2,13 +2,21 @@
 
 supported = on_supported_os
 
-['ansible', 'azure', 'discovery', 'kubevirt', 'openscap', 'remote_execution', 'tasks', 'templates', 'virt_who_configure'].each do |plugin|
+['ansible', 'azure', 'discovery', 'katello', 'kubevirt', 'openscap', 'remote_execution', 'tasks', 'templates', 'virt_who_configure'].each do |plugin|
   describe "foreman::cli::#{plugin}" do
     supported.each do |os, os_facts|
       context "on #{os}" do
         let(:facts) { os_facts }
         let(:pre_condition) { 'include foreman::cli' }
-        let(:plugin_name) { plugin == 'azure' ? 'foreman_azure_rm' : "foreman_#{plugin}" }
+        let(:plugin_name) do
+          if plugin == 'azure'
+            'foreman_azure_rm'
+          elsif plugin == 'katello'
+            'katello'
+          else
+            "foreman_#{plugin}"
+          end
```

Comment:
Feels like a case statement is more natural
```suggestion
          case plugin
          when 'azure'
            'foreman_azure_rm'
          when 'katello'
            'katello'
          else
            "foreman_#{plugin}"
          end
```